nemo/collections/asr/models/rnnt_bpe_models.py

```diff
@@ -196,7 +196,11 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             self.joint.set_wer(self.wer)
 
     def change_vocabulary(
-        self, new_tokenizer_dir: str, new_tokenizer_type: str, decoding_cfg: Optional[DictConfig] = None
+        self,
+        new_tokenizer_dir: str,
+        new_tokenizer_type: str,
+        new_tokenizer_cfg: Optional[DictConfig] = None,
+        decoding_cfg: Optional[DictConfig] = None,
```

Comment:
This API is just confusing. See above comments.

Reply:
And RNNT decoding is wildly different than the vocab changing API so it should not be used like this.

Reply:
Not sure how to improve... the need is to replace a monolingual tokenizer with a multi-lingual one and vice versa so that pretrained monolingual checkpoints could be used to seen multilingual ones

Reply:
made new_tokenizer_dir a Union[str, DictConfig] for now